eng/Versions.props

```diff
@@ -41,6 +41,7 @@
     -->
     <PrivateSourceBuiltSdkVersion>10.0.100-rtm.25523.111</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>10.0.100-rtm.25523.111</PrivateSourceBuiltArtifactsVersion>
+    <DotNet1xxWorkloadManifestVersion>$(MicrosoftNETWorkloadEmscriptenCurrentManifest100100Version)</DotNet1xxWorkloadManifestVersion>
```

Comment:
Do we ever override `MicrosoftNETWorkloadEmscriptenCurrentManifest100100Version` by what's coming in via PSB? Asking as if we do that, the `DotNet1xxWorkloadManifestVersion` property would point at the wrong version as the override happens at a later point.

We in general recommend to not read from other properties in Versions.props due to versions getting overwritten later by PvP.

Reply:
PSB only applies to dependencies in the repos. Since this dependency is at the root level, PSB doesn't influence the version. And even if it did, it should be the correct version anyway as the PSB of a 2xx build should still contain the 1xx version of this package.